README.md

```diff
@@ -0,0 +1,41 @@
+# Multipass Workflows
+This repository contains the workflow definitions for [`multipass launch`](https://multipass.run):
+
+```plain
+$ multipass find
+Image                       Aliases           Version          Description
+# ...
+minikube                                      latest           minikube is local Kubernetes
```

Comment:
Perhaps show faasd as one of the items that comes back in the sample search, to encourage folks to discover and try it?

Reply:
Done.